coalib/results/Diff.py

```diff
@@ -192,6 +192,14 @@ def modified(self):
         return result
 
     @property
+    def has_changes(self):
+        """
+        :return: True if the modified file has real difference when
+                 compared with the original file, False otherwise.
+        """
+        return self.modified != self._file
+
+    @property
```

Comment:
I think it's because you added your changes below line 194; if you put a newline on 193, then your changes starting with the new decorator on 194, you should be good to go.